meta-integrity/recipes-core/initrdscripts/files/init.ima

```diff
@@ -95,20 +95,23 @@ fi
 [ ! -d "$securityfs_dir/ima" ] &&
     print_info "IMA is not enabled. Exiting ..." && exit 2
 
-keyring_id=0x`grep '\skeyring\s*\.ima: ' "${ROOT_DIR}/proc/keys" | awk '{ print $1 }'`
+mount --move ${ROOT_DIR}/proc /proc
 
 # The trusted IMA certificate /etc/keys/x509_evm.der in initramfs was
 # automatically loaded by kernel already. Here is the opportunity to load
 # a custom IMA certificate from the real rootfs.
 for cert in ${ROOT_DIR}/etc/keys/x509_evm*.der; do
     [ ! -s "$cert" ] && continue
+    name=`basename $cert`
+    echo $name
 
-    if ! evmctl import "$cert" "$keyring_id" >"${ROOT_DIR}/dev/null"; then
+    if ! keyctl padd asymmetric "$name" %:.ima < $cert > ${ROOT_DIR}/dev/null; then
         print_critical "Unable to load the custom IMA certificate $cert for IMA appraisal"
```

Comment:
keyutils needs to be added to RDEPENDS_${PN} of initrdscripts-ima.bb.

Reply:
I'll update the list, it's implicit today because ima-evm-utils needs libkeyutils